### PR TITLE
Added ChainX logos & endpoints

### PR DESCRIPTION
**Pull request by qinghuan-chain**

### CODE REVIEW

File: packages/apps-config/src/api/spec/chainx.ts [hunk 1]
```diff
@@ -0,0 +1,446 @@
+/* eslint-disable header/header */
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/chainx.ts:1]:
```suggestion
// Copyright 2017-2021 @polkadot/apps-config authors & contributors
// SPDX-License-Identifier: Apache-2.0

```

File: packages/apps-config/src/api/spec/chainx.ts [hunk 2]
```diff
@@ -0,0 +1,446 @@
...
+/* eslint-disable sort-keys */
+// [object Object]
+// SPDX-License-Identifier: Apache-2.0
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/chainx.ts:4]:
```suggestion

```

File: packages/apps-config/src/api/spec/chainx.ts [hunk 3]
```diff
@@ -0,0 +1,446 @@
...
+export default {
+  Chain: {
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/chainx.ts:6]: I would just suggest adding the `Address` and `LookupSource` types explicitly. The reason is that the defaults are changing on the API layer, see https://github.com/polkadot-js/api/releases/tag/v3.5.1

(One of these types https://polkadot.js.org/docs/api/FAQ#i-cannot-send-transactions-sending-yields-address-decoding-failures)

**Review by jacogr — COMMENTED** [review]
Thank you very much. Some small comments, but looks good otherwise.

File: packages/apps-config/src/api/spec/chainx.ts (hunk 3 above)
Comment (qinghuan-chain) [reply packages/apps-config/src/api/spec/chainx.ts:10]: Ok. Thanks for your advice! it has been repaired.

Comment (jacogr) [reply packages/apps-config/src/api/spec/chainx.ts:10]: No issues, just trying to make sure you don't have pain later :)

**Review by jacogr — APPROVED** [review]
Thank you!

**Pull request merged**